src/internal/ReplaySubject.ts

```diff
@@ -94,8 +117,9 @@ export class ReplaySubject<T> extends Subject<T> {
     return subscription;
   }
 
-  _getNow(): number {
-    return (this.scheduler || queue).now();
+  private _getNow(): number {
+    const { timestampProvider: scheduler } = this;
+    return scheduler ? scheduler.now() : Date.now();
```

Comment:
Is there anything to be gained by keeping the scheduler terminology in here? I mean, it could be a scheduler, but it's entirely possible it could be something else. Why not just use `timestampProvider` as the name for the local const?